packages/web3-eth-contract/test/fixtures/types/web3-v1-contracts/types.ts

```diff
@@ -0,0 +1,64 @@
+/* Autogenerated file. Do not edit manually. */
```

Comment:
Is this file actually generated?

Reply:
It's accidentally committed, was comparing the interface with v1 interfaces.

Reply:
Okay, so `packages/web3-eth-contract/test/fixtures/types/web3-v1-contracts/abi.ts` and `packages/web3-eth-contract/test/fixtures/types/web3-v1-contracts/types.ts` (this file) need to be removed?